src/Components/Modals/AddAdditionalRepositoriesModal.js

```diff
@@ -46,47 +64,49 @@ const AddAdditionalRepositoriesModal = (props) => {
           addSuccessNotification(
             `Repositories have been added for '${keyName}'`
           );
+          setIsSubmitting(false);
         },
         onError: () => {
           addErrorNotification('Something went wrong', {
             description:
               'Your repositories could not be added. Please try again.',
           });
+          setIsSubmitting(false);
+          handleModalToggle();
```

Comment:
Does this change the behavior of the modal? This makes it close on error when I think previously it stayed open on error